vector/src/main/java/im/vector/app/features/home/room/list/RoomListFragment.kt

```diff
@@ -254,6 +296,8 @@ class RoomListFragment @Inject constructor(
         roomListViewModel.handle(RoomListAction.FilterWith(filter))
     }
 
+    private fun resetFilter() = filterRoomsWith("")
```

Comment:
Not sure that it will make more sense to have a dedicated const for this single usage. I would agree if we had this kind of const for the whole project, but we don't have it yet.